cli/tests/pcluster/validators/utils.py

```diff
@@ -21,8 +21,11 @@ def assert_failure_messages(actual_failures, expected_messages):
         if isinstance(expected_messages, str):
             expected_messages = [expected_messages]
         for expected_message in expected_messages:
+            # Either check the regex of expected_message match actual failure or check the whole strings are equal.
+            # This is to deal with strings having regex symbols (e.g. "[") inside
             assert_that(
                 any(re.search(expected_message, actual_failure.message) for actual_failure in actual_failures)
+                or any(expected_message == actual_failure.message for actual_failure in actual_failures)
```

Comment:
We can instead simplify the test by using .* in place of these special  characters. 
We don't need to check the entire error string but that the test is working as expected.